src/plugins/ui_actions/public/service/ui_actions_service.ts

```diff
@@ -110,22 +111,28 @@ export class UiActionsService {
     );
   };
 
-  public readonly getTriggerActions = (triggerId: string) => {
+  public readonly getTriggerActions = (triggerId: string): AnyActionContract[] => {
     // This line checks if trigger exists, otherwise throws.
     this.getTrigger!(triggerId);
 
     const actionIds = this.triggerToActions.get(triggerId);
     const actions = actionIds!
       .map(actionId => this.actions.get(actionId))
-      .filter(Boolean) as Action[];
+      .filter<AnyActionInternal>(Boolean as any)
```

Comment:
```suggestion
      .filter(action => action !== undefined)
```

to avoid the cast to any?

Reply:
Fixed.